collab-document/src/document_data.rs

```diff
@@ -108,3 +111,7 @@ pub fn default_document_collab_data(document_id: &str) -> Result<EncodedCollab,
 pub fn generate_id() -> String {
   nanoid!(10)
 }
+
+pub fn uuid_id_from_document_uuid(document_uuid: &Uuid, key: &str) -> String {
```

Comment:
how about page_id_from_document_id

Reply:
Do you mean to hard-code key = page in this method?